dasharo-compatibility/wifi-bluetooth-support.robot

```diff
@@ -35,7 +39,7 @@ WLE001.001 Wireless card detection (Ubuntu 20.04)
     Login To Linux
     Switch To Root User
     ${out}=    Execute Command In Terminal    lspci | grep "Network controller:"
-    Should Match    ${out}    *${WIFI_CARD_UBUNTU}*
```

Comment:
@filipleple I really do not like this change. You remove the `WIFI_CARD_UBUNTU`, keep the "dead" (not used) variable in configs, possibly also breaking this test for other platforms.

Reply:
@macpijan I've reworked the test to be backwards compatible and run twice for each card

Reply:
> I think we can also have a single entry in the results sheet, if that helps. The PASS would mean both passed. The FAIL would mean that any one of these failed, and would need to be clarified as a comment.

@macpijan cc
@filipleple
This should be the simplest solution.

So leave the original tests.
Add reminder information:
```
Log To Console    Remember to test all variants of wireless cards.
Log    Remember to test all variants of wireless cards.    WARN
```
Add to the information which card was detected.
```
Log To Console    The test passed for the XXX wireless card
Log    The test passed for the XXX wireless card    WARN
```

Of course, these are just suggestions. If you have better ideas, go ahead.

Reply:
Looks good to me.